packages/block-library/src/page-list/editor.scss

```diff
@@ -0,0 +1,11 @@
+.wp-block-navigation {
+	// Block wrapper gets the classes in the editor, and there's an extra div wrapper for now, so background styles need to be inherited.
+	.wp-block-page-list > div,
+	.wp-block-page-list {
+		background-color: inherit;
+	}
+	// Make links unclickable in the editor
```

Comment:
I think the `<Disabled>` component was the previous way of doing this (see the Archives block). Not sure what the pros/cons are of the two ways.

Reply:
Hmm, that won't work here because `Disabled` will prevent any dropdowns from opening. It works for Archives because there are no nesting levels there. It uses the same CSS to achieve the purpose though.